build/run-test-image.sh

```diff
@@ -5,4 +5,4 @@
 set -e
 
 ginkgo -v --slowSpecThreshold=10 test/policy-collection -- -cluster_namespace=$MANAGED_CLUSTER_NAME
-ginkgo -v --slowSpecThreshold=10 test/integration -- -cluster_namespace=$MANAGED_CLUSTER_NAME
+# ginkgo -v --slowSpecThreshold=10 test/integration -- -cluster_namespace=$MANAGED_CLUSTER_NAME
```

Comment:
Could you please delete the line instead? When we want to reenable it, we can always look at the git history to find the command. It's just best practice to not leave commented out code.
```suggestion
```